spec/features/cases/list_filter_journey_spec.rb

```diff
@@ -159,6 +159,13 @@
     scenario 'clearing individual filters', js: true do
       expect(SearchQuery.count).to eq 5
 
+      expect(open_cases_page.search_results_count.text).to eq '1 case found'
+      expect(open_cases_page.filter_crumb_for('FOI - Standard'     )).to be_present
+      expect(open_cases_page.filter_crumb_for('Trigger'            )).to be_present
+      expect(open_cases_page.filter_crumb_for('In time'            )).to be_present
+      expect(open_cases_page.filter_crumb_for('Ready to close'     )).to be_present
+      expect(open_cases_page.filter_crumb_for(@deadline_filter_text)).to be_present
```

Comment:
No need to try and align the code on the right hand side as well, looks odd

Reply:
I agree, was just keeping it consistent with the rest, but happy to introduce different style to this copy/paste/change code